src/main/java/com/iotiq/application/service/SellerService.java

```diff
@@ -62,6 +67,14 @@ public void createSeller(User currentUser) {
     @Transactional
     public void update(SellerUpdateRequest request) {
         Seller currentSeller = getCurrentSeller();
+
+        Set<Skill> removedSkillList = currentSeller.getSkills();
+        removedSkillList.removeAll(request.getSkills());
```

Comment:
If you move this line into the `hasMatchingSkills` method, you can only pass `currentSeller` to it because `removedSkillList` is derived from the `currentSeller`.